### PR TITLE
wording/link/build-instruction updates

### PR DESCRIPTION
**Pull request by MattAtkinson**

### CODE REVIEW

**Comment by abhijit-mahajani-imgtec** [comment]
just minor updates into build-instructions in the commit message

File: creatorkit/Toolbox.md [hunk 1]
```diff
@@ -138,17 +134,29 @@ Initially you need to install dependencies required to build. run the following
...
 <code>$ repo sync</code>
 
+In order to build Contiki applications and flash them to the Clicker boards, you need to install MPLABX IPE and the XC32 compiler from [Microchip](http://microchip.wikidot.com/mplabx:installation). Note that installing the IDE/IPE will link you to the XC32 compiler install instructions once complete. You may need to reboot your machine after installing the compiler.
+
+You can now build everything with the following commands.
+
 <code>$ cd build</code>
 
 <code>$ make</code>
+
```
Comment (abhijit-mahajani-imgtec) [inline creatorkit/Toolbox.md]: might be worth mentioning for more details please refer https://github.com/CreatorKit/build/blob/dev/README.md

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

File: creatorkit/Toolbox.md [hunk 2]
```diff
@@ -25,17 +25,15 @@ There are also 2 ways to power Clicker boards
...
-
-See [building code from source](#building-code-from-source) for information on how to use the source code.
+See [building from source](#building-from-source) for information on how to use the source code.
 
 ### Connecting to Ci40 via Serial
 
 ![serial](../images/ci40serial.jpg)
 
-The next step is to connect to your Ci40 using a serial terminal. Connect a microUSB-USB A cable to the port within the Mikrobus connector and into your PC. You can use any serial terminal program you like set to the standard 115200-N-8-1. The following is an example using miniterm in Ubuntu
+Connect a microUSB-USB A cable to the port within the Mikrobus connector and into your PC. You can use any serial terminal program you like set to the standard 115200-N-8-1. The following is an example using miniterm in Ubuntu
```
Comment (nikhil-zinjurde-imgtec) [inline creatorkit/Toolbox.md]: I'd suggest using microUSB to USB Type-A cable.
Correction: You can use any serial terminal program you like to set the standard 115200-N-8-1.  The following command is an example using miniterm.py on Ubuntu

File: creatorkit/Toolbox.md [hunk 3]
```diff
@@ -138,17 +132,30 @@ Initially you need to install dependencies required to build. run the following
...
 
+Update ~/.bashrc to add repo path in linux path permanently
+
+<code>export PATH=$PATH:~/bin/repo</code>
 
-You should now be able to run the following in a directory of your choice.
+You should now be able to run the following in a directory of your choice to get the code.
 
-<code>$ repo init -u  https://github.com/CreatorKit/manifest.git</code>
+<code>$ repo init -u  https://github.com/CreatorKit/manifest.git -b dev</code>
```
Comment (nikhil-zinjurde-imgtec) [inline creatorkit/Toolbox.md]: I think the branch name can change as per user's choice, so the instruction should use a generic name "-b branch-name"

File: creatorkit/Toolbox.md (hunk 1 above)
Comment (nikhil-zinjurde-imgtec) [inline creatorkit/Toolbox.md]: Yes, this link will be useful to add in here, so that developers will come to know about different make commands as well

**Comment by nikhil-zinjurde-imgtec** [comment]
Please resolve merge conflicts, without this the changes cannot be merged.  PR14 is already merged, so you will need to update the branch, resolve the conflicts and then update this pull request

**Comment by nikhil-zinjurde-imgtec** [comment]
Closing this pull request as this has merge conflicts, please create a new one with these changes along with any additional changes you want to make

**Pull request closed without merging**